tasks/managed/create-pyxis-image/tests/mocks.sh

```diff
@@ -20,7 +20,7 @@ function create_container_image() {
 function cleanup_tags() {
   echo $* >> $(workspaces.data.path)/mock_cleanup_tags.txt
 
-  if [[ "$*" != "--verbose --retry --pyxis-graphql-api https://graphql-pyxis.preprod.api.redhat.com/graphql/ "00?? ]]
+  if [[ "$*" != "--verbose --retry --pyxis-graphql-api https://graphql-pyxis.preprod.api.redhat.com/graphql/ --repository "*" "00?? ]]
```

Comment:
I think it would be worth checking somewhere that the transformation of the REPOSITORY into TARGET_REPOSITORY worked correctly. So either here, but maybe there are two many different strings, so another option is pick one of the tests and add a check of the actual call in mock_cleanup_tags. Some repository string close to real world should be used.

Reply:
I will do. I am changing the PR to draft while I do that to not spam with pR changes.

Reply:
done! PTAL.

Reply:
Looks good, thanks.